### ISSUE TITLE
max line length not enforced for long first line docstrings

### ISSUE THREAD

**Issue by jleverenz** [PyCQA/pycodestyle#622]
Somewhat new to python tooling and pycodestyle, but I believe this should trigger the (default) max line length E501:

```
def foo():
    """Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis pulvinar vitae

    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    """
```

The first docstring line is 83 characters (> 79). I believe pycodestyle may be looking at the tokenized docstring, which doesn't include the leading whitespace in front of the opening """ in the physical line.

**Comment by asottile** [comment]
Interesting, while attempting to write a test for this it fails to reproduce when added to `testsuite/E50.py`, in order to trigger a failure I needed to make a new file `testsuite/E501.py` -- I suspect something is wonky with the test suite.

**Comment by asottile** [comment]
Interesting.

```python
def test():
    """fooooooooooooooooooooooooooooooooooooooooooooo ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo
    bar
    """
```

triggers E501

but 

```python
def test():
    """foooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo
    bar
    """
```

does not

**Comment by asottile** [comment]
Oh that's the special case for "urls", I'll proceed with the testcase in the ticket
